tests/integration/hosts/maya/input/startup/userSetup.py

```diff
@@ -0,0 +1,28 @@
+import logging
+import sys
+
+from maya import cmds
+
+import pyblish.util
+
+
+def setup_pyblish_logging():
+    log = logging.getLogger("pyblish")
+    hnd = logging.StreamHandler(sys.stdout)
+    fmt = logging.Formatter(
+        "pyblish (%(levelname)s) (line: %(lineno)d) %(name)s:"
+        "\n%(message)s"
+    )
+    hnd.setFormatter(fmt)
+    log.addHandler(hnd)
```

Comment:
I don't see a reason we need to abbreviate the variables here
```suggestion
    log = logging.getLogger("pyblish")
    handler = logging.StreamHandler(sys.stdout)
    formatter = logging.Formatter(
        "pyblish (%(levelname)s) (line: %(lineno)d) %(name)s:"
        "\n%(message)s"
    )
    handler.setFormatter(formatter)
    log.addHandler(handler)
```